Contributors flow improvment (#25)
testimio/root-cause: CONTRIBUTING.md

# Contributing Guide 

## Signing the CLA

In order to contribute code you need to sign [the CLA](https://docs.google.com/document/d/1X-EoFIW1PMe8H-DJyZDScsTolPKm22-9P1JgSuiFNfE/edit?usp=sharing) which makes it possible for us to release this code as AGPL but consume it internally in a different license.

If you have any issues with the CLA or signing it please contact us (benji@testim.io / bnaya@testim.io) directly.

## Technical Know-How

This monorepo is managed with yarn workspaces.  

> The codebase is written with typescript, and linted using eslint.  

Reviewer: ```suggestion
The codebase is written with TypeScript, and linted using ESLint.  
```
?

> It's recommended to have properer IDE integration with the tools in use (eslint, editorconfig) before modifying the code

Reviewer: ```suggestion
It's recommended to have proper IDE integration with the tools in use (eslint, editorconfig) before modifying the code
```

### Prerequisite
- yarn [Installation instructions](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
- node 12/14 [Installation instructions](https://nodejs.org/en/download/package-manager/)

> ## Dependencies install

Reviewer: ```suggestion
## Installing Dependencies
```

> The first step for local development would be to install the dependencies

Reviewer: ```suggestion
The first step for local development would be to install dependencies:
```

```sh
# install dependencies
yarn
```

## Running the tests
To run the whole test suite:
```sh
yarn test
```

> On a clean clone, the test are expected to pass. if they don't, please let us know!

Reviewer: ```suggestion
On a clean clone, the tests are supposed to pass. if they don't, please let us know!
```
